.rubocop.yml

```diff
@@ -4,6 +4,8 @@
 require:
   - rubocop-performance
   - rubocop-rails
+  - rubocop-rake
```

Comment:
Out of curiosity what pushed you to include these along with 2.7.2?

Reply:
When I ran rubocop after updating (to check for new cops, mostly), it gave me a suggestion to include those, which I hadn't seen before. I figured including them would both be a good idea in general and make the build cleaner, so I went ahead and did.

Reply:
Oh, that's really neat. Love it.